mne_bids/tests/test_mne_bids.py

```diff
@@ -109,6 +110,42 @@ def test_fif():
     assert op.exists(op.join(output_path, 'participants.tsv'))
 
 
+def test_fif_processed():
+    """Test raw_to_bids conversion for processed Neuromag data."""
+    output_path = _TempDir()
+    data_path = testing.data_path()
+    # sss processed file
+    raw_fname = op.join(data_path, 'SSS', 'sample_audvis_trunc_raw_sss.fif')
+    raw = mne.io.read_raw_fif(raw_fname)
+    write_raw_bids(raw, bids_basename, output_path, overwrite=False)
+    # ensure the sidecar json file has processing info
+    data_meta_fname = make_bids_basename(
+        subject=subject_id, session=session_id, task=task, run=run,
+        acquisition=acq, processing='sss', suffix='%s.json' % 'meg',
+        prefix=op.join(output_path, 'sub-01/ses-01/meg'))
```

Comment:
why not use plain python functions here?

like `op.join('sub-01', 'ses-01', 'meg', bids_basename + '_meg.json')`

It looks far simpler to me.

Reply:
oh, well before we didn't have `bids_basename` and I pretty much just ported the old code to the new functions without thinking about making a change like that sorry.
I agree you snippet is definitely nicer :)